README.md

```diff
@@ -62,14 +64,10 @@ replace it with the proper embed-code.
   - [Twitch](#twitch)
   - [Twitter](#twitter)
   - [YouTube](#youtube)
-- [Options](#options-1)
+- [Options](#options)
   - [customTransformers](#customtransformers)
-    - [Properties](#properties)
-      - [`getHTML(url, options)`](#gethtmlurl-options)
-      - [`name`](#name)
-      - [`shouldTransform(url)`](#shouldtransformurl)
-    - [Example transformer](#example-transformer)
```

Comment:
This was removed when I used the `doctoc` functionality built-into kcd-scripts. Let me know if you'd like me to revert this change and add the remark plugin link by hand instead.